test/plug/router_test.exs

```diff
@@ -317,6 +360,20 @@ defmodule Plug.RouterTest do
     end
   end
 
+  test "assigns path params to conn params" do
+    assert true == true
```

Comment:
I was talking about `assert true == true`, which you have cleaned up here but not in the next test btw 😄 But that was it yeah!